src/lib/elements/table/cell.svelte

```diff
@@ -14,3 +14,9 @@
     role="cell">
     <slot />
 </div>
+
+<style lang="scss">
+    .table-col {
+        position: relative;
+    }
+</style>
```

Comment:
Why not `u-position-relative`? Did you check if this doesn't interfere with dropdowns and tooltips?